examples/basic/version.tf

```diff
@@ -1,12 +1,12 @@
 terraform {
-  required_version = ">= 1.3.0, <1.7.0"
+  required_version = ">= 1.4.0, <1.7.0"
 
   # Ensure that there is always 1 example locked into the lowest provider version of the range defined in the main
   # module's version.tf (usually a basic example), and 1 example that will always use the latest provider version.
   required_providers {
     ibm = {
       source  = "IBM-Cloud/ibm"
-      version = "1.49.0"
+      version = "1.56.1"
```

Comment:
the lowest range you defined in version.tf is `1.49.0` - so basic example should lock into that (or else bump the minimum version in version.tf)